sales_page/index.html

```diff
@@ -6,20 +6,20 @@
 </head>
 <body>
   <div id="sales">
-    <article
+    <article>
       <form id="addSalesForm">
         <fieldset>
-          <legend>Sales information</legend>
-          <lable>Location</label>
-            <input location = "location" type = "text"/>
-          <label>Avg cookies</label>
-            <input avgCookies = "avgCookies" type = "text"/>
-          <label>min Customer</label>
-            <input minCustomer = "minCustomer" type = "text"/>
-          <label>max Customer</label>
-            <input maxCustomer = "maxCustomer" type = "text"/>
+            <legend>Sales information</legend>
+            <lable>Location</label>
+            <input name = "location" type = "text"/>
```

Comment:
Sorry, I think i misunderstood your point. Pls ignore my previous message.  I revised the code as suggested. Thanks.